doctr/models/classification/vit/pytorch.py

```diff
@@ -109,7 +120,7 @@ def _vit(
     return model
 
 
-def vit(pretrained: bool = False, **kwargs: Any) -> VisionTransformer:
+def vit_b(pretrained: bool = False, **kwargs: Any) -> VisionTransformer:
     """VisionTransformer architecture as described in
```

Comment:
I suggest specifying the version of the archi in the docstring as well "VisionTransformer-B"